src/Draco.Compiler.Tests/EndToEnd/EndToEndTestsBase.cs

```diff
@@ -35,22 +40,33 @@ protected static Assembly Compile(string sourceCode)
         string methodName,
         TextReader? stdin,
         TextWriter? stdout,
+        string moduleName = "FreeFunctions",
```

Comment:
We might want to factor out the free functions module name into some global constant, as we have it littered in quite a few places already